Customizing/global/plugins/Services/Repository/RepositoryObject/ReportWBDPoints/classes/class.ilObjReportWBDPoints.php

```diff
@@ -137,27 +130,37 @@ protected function fetchData(callable $callback) {
 				  ."     ON usrcrs.crs_id = crs.crs_id\n"
 				  ."         AND crs.hist_historic = 0\n"
 				  ." WHERE usrcrs.hist_historic = 0\n"
-				  ."     AND usrcrs.wbd_booking_id IS NOT NULL\n";
+				  ."     AND usrcrs.wbd_booking_id IS NOT NULL\n"
+				  ."     AND usr.hist_historic = 0\n"
+				  ."     AND crs.hist_historic = 0\n";
 
 		$filter = $this->filter();
 		if($this->filter_settings) {
 			$settings = call_user_func_array(array($filter, "content"), $this->filter_settings);
 			$to_sql = new \CaT\Filter\SqlPredicateInterpreter($db);
-			$dt_query = $to_sql->interpret($settings[0]["period_pred"]);
+			$dt_query = $to_sql->interpret($settings["period_pred"]);
 			$query .= "     AND ".$dt_query;
```

Comment:
Bist du dir sicher das die [0] hier überall weg muss? In den anderen Berichten ist sie noch da.